docs/configuration/http_conn_man/header_sanitizing.rst

```diff
@@ -12,6 +12,7 @@ depends on the :ref:`use_remote_address <config_http_conn_man_use_remote_address
 
 Envoy will potentially sanitize the following headers:
 
+* :ref:`x-envoy-decorator-operation <config_http_filters_router_x-envoy-decorator-operation>`
```

Comment:
I think this needs to be implemented? (Remove header on external requests, need to add test for that also in conn_manager_utility)